pom.xml

```diff
@@ -1339,5 +1339,26 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>disableWizard</id>
+      <activation>
+        <property>
+          <name>!enableWizard</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jenkins-ci.tools</groupId>
+            <artifactId>maven-hpi-plugin</artifactId>
+            <configuration>
+              <systemProperties>
+                <jenkins.install.state>TEST</jenkins.install.state>
```

Comment:
Yes, it's a [better option](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/jenkins/install/InstallUtil.java#L163)